test/MOL/MOL_1D_Linear_Diffusion.jl

```diff
@@ -59,6 +59,37 @@ using ModelingToolkit: Differential
             @test all(isapprox.(u_approx, exact, atol=0.01))
         end
     end
+    
+    @parameters t x
+    
+    # BCs with 2nd order Neumann
+    bcs = [u(0,x) ~ cos(x),
+           Dxx(u(t,0)) ~ -exp(-t),
+           Dxx(u(t,Float64(π))) ~ exp(-t)]
```

Comment:
You can't specify a 2nd order BC on a 2nd order system. See https://github.com/SciML/DiffEqOperators.jl/pull/382#issuecomment-828782462

Reply:
Yes, this was just to show that desired operation gets carried out safely and in correct manner. 
But yeah, I think a higher ordered eqn would be good.